### PR TITLE
submit documentation

### PR DESCRIPTION
**Pull request by 12mihai05**
### Pull Request Overview

This pull request adds the Project Documentation for the Music Player.

### TODO or Help Wanted

This pull request contains my project's documentation.
Further modifications of the documentation will be uploaded here.

### Author

Signed-off-by: Pasaroiu Mihai Octavian <pasaroiumihai@yahoo.com>

### CODE REVIEW

File: docs/project/2024/mihai.pasaroiu/index.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+| MicroSD Card Module | Reads the song from the microSD| [4.39 RON](https://www.optimusdigital.ro/ro/componente-electronice-rezistoare/33-rezistoare-set.html) |
+
+
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |
```
Comment (alexandruradovici) [inline docs/project/2024/mihai.pasaroiu/index.md:75]: Do you need these libraries?

File: docs/project/2024/mihai.pasaroiu/index.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+## Links
+
+<!-- Add a few links that inspired you and that you think you will use for your project -->
+
+1. [link](https://example.com)
+2. [link](https://example3.com)
```
Comment (alexandruradovici) [inline docs/project/2024/mihai.pasaroiu/index.md:82]: Please add some links for your project.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]
Please fill the pull request template.

File: docs/project/2024/mihai.pasaroiu/Arhitecture.png
Comment (alexandruradovici) [inline docs/project/2024/mihai.pasaroiu/Arhitecture.png:1]: The spelling is wrong for this file name :)

**Review by alexandruradovici — CHANGES_REQUESTED** [review]
Please fill the pull request template.

**Review by alexandruradovici — APPROVED** [review]

**Review by DTeodor-Alexaandru — APPROVED** [review]

**Pull request merged**